layouts/soarian_dstu2_sidebar.html

```diff
@@ -0,0 +1,72 @@
+<div id="js-sidebar" class="sidebar-shell">
+  <div class="js-toggle-list sidebar-module expandable">
+    <ul>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/soarian/ss/">Overview</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/soarian/dstu2/soarian_authorization/">Authorization</a></li>
+		   <li><a href="/soarian/dstu2/conformance/">Conformance Metadata</a></li>
+        </ul>
+      </li>  
+		<!-- <li class="js-topic">
```

Comment:
This can stay for now since we'll be pushing in the remaining changes within the week